### PR TITLE
Add Support For CyHy Notifications

### PR DESCRIPTION
**Pull request by dav3r**
This PR enables cyhy-mailer to send out CyHy notification emails.

### CODE REVIEW

**Review by felddy — APPROVED** [review]
Looking good. But it's late.  Lots of things look good.  Oooh f{strings}!  Nice.  
And tests!  
ZZzzz...

File: Dockerfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ RUN mkdir ${MAILER_HOME} \
 RUN apk --no-cache del shadow
 
 # Prepare to Run
-WORKDIR MAILER_HOME
+WORKDIR ${MAILER_HOME}
```
Comment (jsf9k) [inline Dockerfile:30]: Good eye, sir!

File: README.md [hunk 2]
```diff
@@ -29,71 +29,62 @@ pip install git+https://github.com/cisagov/cyhy-mailer.git
 Usage:
   cyhy-mailer report [--cyhy-report-dir=DIRECTORY]
 [--tmail-report-dir=DIRECTORY] [--https-report-dir=DIRECTORY]
-[--cybex-scorecard-dir=DIRECTORY] [--mail-server=SERVER] [--mail-port=PORT]
-[--smtp-user=SMTP_USER] [--smtp-password=SMTP_PASS] [--db-creds-file=FILENAME]
-[--smtp-creds-file=FILENAME] [--batch-size=SIZE] [--summary-to=EMAILS] [--debug]
+[--cybex-scorecard-dir=DIRECTORY] [--cyhy-notification-dir=DIRECTORY]
```
Comment (jsf9k) [inline README.md:32]: Good eye again!

**Review by jsf9k — APPROVED** [review]
Strong work!

**Pull request merged**